cmd/determine-rebottle-runners.rb

```diff
@@ -48,10 +48,13 @@ def determine_rebottle_runners
         if macos_version.outdated_release? || macos_version.prerelease?
           nil
         else
-          macos_runners = [{
-            runner: "#{macos_version}-#{ENV.fetch("GITHUB_RUN_ID")}-#{ENV.fetch("GITHUB_RUN_ATTEMPT")}",
-          }]
-          macos_runners << { runner: "#{macos_version}-arm64" } if macos_version >= :big_sur
+          ephemeral_suffix = "-#{ENV.fetch("GITHUB_RUN_ID")}-#{ENV.fetch("GITHUB_RUN_ATTEMPT")}"
```

Comment:
Can be done in a separate PR, but do we need `GITHUB_RUN_ATTEMPT`? This forces a restart for all ephemeral runners even if you only wanted to restart the failing ones.

Reply:
I'll look into this, yes. It will require ci-orchestrator changes but I'll figure out something.